jobserver/models/core.py

```diff
@@ -782,6 +783,11 @@ class UserManager(BaseUserManager.from_queryset(UserQuerySet), BaseUserManager):
     pass
 
 
+def human_memorable_token(size=8):
```

Comment:
tiny nit: as I understandable, it's for reading & transcribing not memorisation

```suggestion
def human_readable_token(size=8):
```

Reply:
I specifically called it memorable because the human cannot look it at the same time they are typing it, due to how the VPN works. They may write it down, but I wanted to make it clear that we want to be able for a human to potentially hold in memory whilst they switch to VPN/RDP.